CMakeLists.txt

```diff
@@ -46,6 +46,14 @@ option(IREE_BUILD_DEBUGGER "Builds the IREE debugger app." OFF)
 option(IREE_BUILD_PYTHON_BINDINGS "Builds the IREE python bindings" OFF)
 option(IREE_BUILD_JAVA_BINDINGS "Builds the IREE java bindings." OFF)
 option(IREE_BUILD_EXPERIMENTAL "Builds experimental projects." OFF)
+option(IREE_ENABLE_CCACHE "Use ccache if installed to speed up rebuilds." OFF)
+
+if (${IREE_ENABLE_CCACHE})
```

Comment:
can you move if() down in the file to someplace else we set things like this? it's nice to have all the options a human needs to read up in one place at the top of the file so you don't have to go hunting for them

Reply:
Actually the options are already grouped in different sections, and I had the wrong section. Moved.